Source/Android/jni/MainAndroid.cpp

```diff
@@ -193,7 +193,7 @@ static bool LoadBanner(std::string filename, u32* Banner)
 
   if (pVolume != nullptr)
   {
-    int Width, Height;
+    u32 Width, Height;
```

Comment:
Borderline outside the scope of this PR, but this could use some style treatment. Ignorable comment if you want it in another cleanup PR.

Reply:
These functions for getting volume metadata are bad for more serious reasons anyway. I'm planning to make a large overhaul of the Android game list code in the future, so that would take care of the variable naming.